### DISCUSSION TITLE
Forms Package: How I could add an icon to validation error message?

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#8926] [Help]
something like this:

<img width="722" alt="Screenshot 2023-10-06 at 14 41 38" src="https://github.com/filamentphp/filament/assets/121255432/9b9aecd3-166b-4a25-a2dd-11f7550c2c6b">

**Comment by leandrocfe** [comment]
A [workaround](https://github.com/filamentphp/filament/discussions/8373#discussioncomment-6957597)

**Reply by devhoussam1998** [reply]
this will not add an icon near the validation text
